### PR TITLE
New distros

### PR DESCRIPTION
**Pull request by flichtenheld**

### CODE REVIEW

File: buildbot-host/buildbot-worker-fedora-36/env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+BUILDMASTER=buildmaster
+WORKERNAME=fedora-35
```
Comment (mattock) [inline buildbot-host/buildbot-worker-fedora-36/env]: Should be _fedora-36_.

**Comment by mattock** [comment]
There was one typo which I commented on. Otherwise looks good except for the merge conflict.

**Comment by mattock** [comment]
Btw. previously there was a bug in Ubuntu 20.04 (our Docker host) that prevented launching Fedora 36 containers. What made that issue go away?

**Comment by flichtenheld** [comment]
> Btw. previously there was a bug in Ubuntu 20.04 (our Docker host) that prevented launching Fedora 36 containers. What made that issue go away?

I don't know. Clearly someone fixed it, it was a while ago.

**Comment by flichtenheld** [comment]
> There was one typo which I commented on. Otherwise looks good except for the merge conflict.

Okay, will rebase

File: buildbot-host/buildbot-worker-fedora-36/env (hunk 1 above)
Comment (flichtenheld) [reply buildbot-host/buildbot-worker-fedora-36/env]: Fixed

**Pull request merged**